app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -11,18 +11,111 @@
 @import "shop-taxon-flag";
 @import "shop-popovers";
 
+$sidebar-small-width: 85%;
+$sidebar-medium-width: 65%;
+$sidebar-large-width: 45%;
+$sidebar-footer-height: 5em;
+
 .darkswarm {
+  .shop-filters-sidebar {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    .background {
+      position: fixed;
+      top: 0;
+      right: 0;
+      z-index: 200;
+      height: 100%;
+      width: 100%;
+      background-color: rgba(0, 0, 0, 0.15);
```

Comment:
shouldn't this be tracked in a var and come from the design mockups?

Reply:
This was a bit of a deviation form the mockups, as the original used a CSS blur effect instead, but I couldn't get it to work and it didn't seem to have wide browser support. I'll extract it to a variable though :+1: